iframe/priv/static/iframe/v5.html

```diff
@@ -0,0 +1,240 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <base target="_parent">
```

Comment:
When setting this you don't need to set <a target="_parent" />
I'm not sure if this is enough to ship a new iframe version. Maybe keep this in mind when there are other changes and for now you have to write target="_parent" on each link in an iframe?

Reply:
It's fine, we don't change the iframes often anyway :)